docs/directives.md

```diff
@@ -2,16 +2,25 @@
 
 ```js static
 import { Tooltip } from '@nextcloud/vue'
-
-Vue.directive('tooltip', Tooltip)
 ```
 
 The tooltip directive is based on v-tooltip. For a full feature list see the projects [README](https://github.com/Akryum/v-tooltip/blob/master/README.md#directive)
 
 In the template, use the `v-tooltip` directive:
 
 ```vue
-<a v-tooltip="'You have new messages.'">Hover me</a>
+    <template>
```

Comment:
I couldn't figure out how to globally import the `tooltip` directive for the styleguide, so I individually imported it for the examples.

Reply:
Maybe that's what may help?
https://vue-styleguidist.github.io/Configuration.html#enhancepreviewapp

Reply:
Yes, thanks a lot. That's what I was looking for.